frontend/common/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  DotsHorizontalIcon,
+} from "@heroicons/react/solid";
+import * as React from "react";
+import { useIntl } from "react-intl";
+import { Button } from "..";
+import { DOTS, usePagination } from "./usePagination";
+
+export interface PaginationProps {
+  /** The current active page. */
+  currentPage: number;
+  /** Total count of items from array. */
+  totalCount: number;
+  /** The minimum number of page buttons to be shown on each side of the current page button. */
+  siblingCount: number;
+  /** The maximum number of visible items on a single page. */
+  pageSize: number;
+  /** Callback that changes to the page number value. */
+  handlePageChange: (pageNumber: number) => void;
+}
+
+const Pagination: React.FunctionComponent<PaginationProps> = ({
+  totalCount,
+  siblingCount,
+  currentPage,
+  pageSize,
+  handlePageChange,
+}) => {
+  const intl = useIntl();
+  const paginationRange = usePagination({
+    currentPage,
+    totalCount,
+    siblingCount,
+    pageSize,
+  });
+
+  // If there are less than 2 times in pagination range we shall not render the component
+  if (currentPage === 0 || paginationRange.length < 2) {
+    return null;
+  }
+
+  const nextPage = () => {
+    handlePageChange(currentPage + 1);
+  };
+
+  const previousPage = () => {
+    handlePageChange(currentPage - 1);
+  };
+
+  const lastPage = paginationRange[paginationRange.length - 1];
+  const isLeftArrowDisabled = currentPage === 1;
+  const isRightArrowDisbaled = currentPage === lastPage;
```

Comment:
Should be `isRightArrowDisabled`

Reply:
Ah, my Code Spell Checker extension wasn't added to my WSL env. Nice catch 👍